src/shared/components/mutationTable/column/SampleColumnFormatter.tsx

```diff
@@ -76,24 +76,30 @@ export default class SampleColumnFormatter
         }
     }
 
-    public static renderFunction(data:Mutation[])
+    public static renderFunction(data:Mutation[], studyId?: string)
     {
         const sampleId:string = SampleColumnFormatter.getTextValue(data);
         const text:string = SampleColumnFormatter.getDisplayValue(data);
         const toolTip:string = SampleColumnFormatter.getTooltipValue(sampleId);
-        const linkToPatientView:string = "#"; // TODO generate or get it from somewhere else
 
-        let content = (
-            <a href={linkToPatientView} target='_blank'>
-                <span className={styles['text-no-wrap']}>{text}</span>
-            </a>
-        );
+        let content = <span className={styles['text-no-wrap']}>{text}</span>;
```

Comment:
it's probably not necsesary to do this with css module.  The classname implies exactly what this does, so it could just be a global class the we reuse elsewhere.